config/initializers/ahoy.rb

```diff
@@ -1,6 +1,5 @@
 class Ahoy::Store < Ahoy::DatabaseStore
 end
 
-Ahoy.visit_duration = 30.minutes
 # set to true for JavaScript tracking
```

Comment:
please keep Ahoy.visit_duration and introduce Setting.visit_duration to load from ENV('VISIT_DURATION_MINUTE), use default 30 if it's not present